packages/dx-react-grid/src/components/group-panel-layout.tsx

```diff
@@ -1,11 +1,23 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
 import { findDOMNode } from 'react-dom';
 import { DropTarget } from '@devexpress/dx-react-core';
 import { getGroupCellTargetIndex } from '@devexpress/dx-grid-core';
 import { ItemLayout } from './group-panel-layout/item-layout';
+import { GroupingPanel as GP } from '../types';
+
+// tslint:disable-next-line: max-line-length
+class GroupPanelLayoutBase extends React.PureComponent<GP.LayoutProps, GP.GroupingItemLayoutState> {
+  static defaultProps: Partial<GP.LayoutProps>;
+  handleDragEvent: (...args: any) => void;
+  onEnter: (any) => void;
+  onOver: (any) => void;
+  itemRefs: any;
+  draggingColumnName?: string;
```

Comment:
Wouldn't it be better to declare it as `draggingColumnName: string | null = null`. There is no undefined state for this field - it is either set to some string during dragging or `null` otherwise.